scripts/lib/install

```diff
@@ -24,21 +27,25 @@ EOF
 
 # Shell option parsing.  Over time, we'll want to move some of the
 # environment variables below into this self-documenting system.
-args="$(getopt -o '' --long help,no-init-db,no-dist-upgrade,no-overwrite-settings,self-signed-cert,certbot,postgres-missing-dictionaries,remote-postgres,hostname:,email:,cacert: -n "$0" -- "$@")"
+args="$(getopt -o '' --long help,hostname:,email:,certbot,cacert:,self-signed-cert,postgres-missing-dictionaries,remote-postgres,no-init-db,no-overwrite-settings,no-dist-upgrade -n "$0" -- "$@")"
```

Comment:
At least `cacert` and `self-signed-cert` have inverted order here vs. above.  Was the intent to make the ordering match precisely?

Reply:
Yup -- thanks for the catch.